dali/operators/audio/mel_scale/mel_filter_bank.cc

```diff
@@ -23,8 +23,9 @@ DALI_SCHEMA(MelFilterBank)
     .DocStr(R"code(Converts a spectrogram to a mel spectrogram by applying a bank of
 triangular filters.
 
-Expects an input with at least 2 dimensions where the last two dimensions correspond to
-the fft bin index and the window index, respectively.
+Expects an input with at least 2 dimensions.
```

Comment:
This should not be a requirement - I mean, you can just calculate a single Mel *spectrum* (not necessarily a spectrogram).
It can be worked around by adding artificial dimensions for use by the kernels.
If, on the other hand, there are more than 2 dimensions, all trailing and leading dimensions can be collapsed.
Can we even handle a case when `f` is somewhere in the middle - think, `AfB` layout?

Reply:
The GPU kernel supports any f axis and the operator